docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -611,7 +610,9 @@ The table below describe all the available configuration options.
 | quarkus.kubernetes.liveness-probe                  | Probe                                     |             | ( see Probe )
 | quarkus.kubernetes.readiness-probe                 | Probe                                     |             | ( see Probe )
 | quarkus.kubernetes.sidecars                        | Map<String, Container>                    |             |
-| quarkus.kubernetes.expose                          | boolean                                   |             | false
```

Comment:
We might want to keep this around for backwards compatibility reasons.
Don't have a very strong opinion about it. It`s what we use to do in related cases.

Reply:
Never mind, just noticed that its only removed from the documentation.